crates/cxx-qt-gen/test_outputs/passthrough_and_naming.rs

```diff
@@ -85,6 +86,7 @@ pub mod ffi {
         #[namespace = "cxx_qt::multi_object"]
         type MyObjectRust;
     }
+    impl cxx_qt::Upcast<ffi::QStringListModel> for ffi::MyObject {}
```

Comment:
this needs to end up outside of the bridge, so must be inserted into the wrong list...

Reply:
(see other comment about `cxx_qt_mod_contents` vs `cxx_mod_contents`)